app/manifest.json

```diff
@@ -45,11 +46,13 @@
         "options.html"
     ],
     "permissions": [
-        "declarativeContent",
-        "activeTab",
```

Comment:
Is `activeTab` Chrome specific?

Reply:
It's not, however Chrome and Firefox implement the permission differently. ActiveTab in Firefox doesn't give you access to the URL while it does in Chrome. Tabs gives access to the URL in both.